frontend/src/components/header/Header.js

```diff
@@ -1,56 +1,59 @@
-import React, {Component} from 'react';
-import {Grid} from '@material-ui/core';
-import logo from '../../assets/ethBuenosAires.png';
+import React, { Component } from 'react'
+import { Grid } from '@material-ui/core'
+import logo from '../../assets/ethBuenosAires.png'
+import logoLight from '../../assets/ethBuenosAires-light.svg'
 import { Link } from 'react-router-dom'
```

Comment:
As a good practice try to keep the dependencies on the top and separated by a space the app dependencies.

Example:
```javascript
import React, { Component } from 'react';
import {Grid} from '@material-ui/core';
import { Link } from 'react-router-dom';

import logo from '../../assets/ethBuenosAires.png';
import logoLight from '../../assets/ethBuenosAires-light.svg'
```